packages/adapter-node/package.json

```diff
@@ -45,10 +45,11 @@
 		"@polka/url": "^1.0.0-next.28",
 		"@sveltejs/kit": "workspace:^",
 		"@sveltejs/vite-plugin-svelte": "^5.0.1",
-		"@types/node": "^18.19.48",
+		"@types/node": "^20.17.16",
```

Comment:
Bumping node types from 18 to 20 seems to fix some rollup/vite types issue because that's what they're using as well I think.

Reply:
This bump is no longer needed after we fixed the TypeScript issue with crossws types